cmd/eskip/load_test.go

```diff
@@ -142,6 +142,20 @@ func TestCheckRepeatingRouteIds(t *testing.T) {
 	}
 }
 
+func TestCheckEmptyBackend(t *testing.T) {
+	const name = "testFile"
+	err := withFile(name, `foo: Method("POST") -> "https://www.example1.org";empty: Method("POST") -> "";`, func(_ *os.File) {
+		err := checkCmd(cmdArgs{in: &medium{typ: file, path: name}})
+		if err == nil {
+			t.Error("Expected an error for empty backend")
+		}
```

Comment:
Lets check specific error value here.

Reply:
Do you mean something like `if err == nil || !errors.Is(err, expectedError) ` ?

Reply:
No, just something like `if err == nil || err.Error() != "route has empty backend: empty"`